GameServerLib/Logic/Game.cs

```diff
@@ -255,6 +257,15 @@ private bool HandleDisconnect(Peer peer)
                 {
                     PacketNotifier.NotifyUnitAnnounceEvent(UnitAnnounces.SummonerDisconnected, peerinfo.Champion);
                 }
+                _logger.LogCoreInfo("somebody disconnected");
```

Comment:
This can have more details (like clientId and/or summoner name) and should be grammatically correct.